### DISCUSSION TITLE
API returns albums and singles

### DISCUSSION THREAD

**Discussion by smoothquark** [tidal-music/discussions#98] [General]
It appears that the albums endpoint returns both albums and singles? Is there any way to differentiate between them and only return "official" albums? I thought about filtering out "numberOfItems": 1, and this will filter out the obvious singles, but how do I filter out for example album id 386158184 (Edge of Saturday Night) which has 2 tracks?

**Comment by aliaksei-taliuk** [comment]
Hi @smoothquark 
Thanks for reaching out to us here. We will discuss how/if we should approach these situations and get back to you as soon as we have a solution/answer. 
Best reagards

**Comment by smoothquark** [comment]
Thank you.

**Reply by aliaksei-taliuk** [reply]
@smoothquark Hi. We have added a new property to the album's "attributes" section called "type". It's an enumeration of 3 possible values: EP, ALBUM, SINGLE. Hope it addresses your question.

**Comment by smoothquark** [comment]
Ah. Fantastic. Thank you. I shall have a play with it.
